function-reference/navigateTo.md

```diff
@@ -15,7 +15,8 @@ navigateTo(pageName: string) -> void
 
 | **Argument Name** | **Description** |
 | :--- | :--- |
-| **pageName** |  Page name to which we would like to be transported. It's case sensitive. |
+| **pageNameOrUrl** |  Page name or URL to which we would like to be transported. PageName is case sensitive. |
+| **params** |  Query params, to share information with destination. |
```

Comment:
Sorry, I just noticed. Since we are adding the `params` to the Arguments section here, can we also update the `Signature` section to reflect this? Else, the arguments is out of sync with the function signature.